documentation/book/con-kafka-connect-authentication.adoc

```diff
@@ -48,7 +48,7 @@ To use the authentication using the SCRAM-SHA-512 SASL mechanism, set the `type`
 SCRAM-SHA-512 uses a username and password to authenticate.
```

Comment:
_Please re-word this concept module as follows:_

### SCRAM-SHA-512 authentication

To configure Kafka Connect to use SCRAM-SHA-512 SASL authentication, set the `type` property to `scram-sha-512`. This authentication mechanism requires a username and password.

* Specify the username in the `username` property.
* In the `password` property, specify a link to a `Secret` containing the password in the `passwordSecret` property.
* In the `secretName` property, specify the name of the `Secret` and the key under which the password is stored inside the `Secret` in the `password` property. 

#### Example of SCRAM-SHA-512 client authentication configuration

[No changes to the code sample are needed.]